pyteal/ast/abi/array_dynamic.py

```diff
@@ -100,11 +100,34 @@ def length(self) -> Expr:
 DynamicArray.__module__ = "pyteal.abi"
 
 
+class DynamicBytesTypeSpec(DynamicArrayTypeSpec):
+    def __init__(self) -> None:
+        super().__init__(ByteTypeSpec())
+
+    def new_instance(self) -> "DynamicBytes":
+        return DynamicBytes()
+
+    def annotation_type(self) -> "type[DynamicBytes]":
```

Comment:
```suggestion
    def annotation_type(self) -> type["DynamicBytes"]:
```

Reply:
I note the other annotation_type above has the same quote around "type[DynamicArray[T]]", if the above change is correct the other one should be tweaked as well